crd2pulumi-crontabs/README.md

```diff
@@ -0,0 +1,233 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Generating CronTab CustomResources with `crd2pulumi`
+
+This example generates a strongly-typed CronTab CustomResource from the k8s CRD specified in `resourcedefinition.yaml` in TypeScript and Go. Afterwards, we'll use this generated code with Pulumi to deploy the CRD and create an instance. For more documentation on `crd2pulumi`, check out the [project's GitHub page](https://github.com/pulumi/pulumi-kubernetes/tree/master/provider/cmd/crd2pulumi).
+
+## Setting up crd2pulumi
+We'll set up `crd2pulumi` by downloading the latest binary. If you're interested in building `crd2pulumi` itself, you can find instructions in the project [README](https://github.com/pulumi/pulumi-kubernetes/tree/master/provider/cmd/crd2pulumi).
+
+You can find the download links for the [latest binaries on GitHub](https://github.com/pulumi/pulumi-kubernetes/releases/tag/crd2pulumi/v1.0.0). For this example we're using `darwin-amd64`, so if you're using a different OS, make sure to use the correct download link.
+```bash
+$ wget https://github.com/pulumi/pulumi-kubernetes/releases/download/crd2pulumi%2Fv1.0.0/crd2pulumi-darwin-amd64.tar.gz
+$ tar -xvf crd2pulumi-darwin-amd64.tar.gz
+$ mv ./releases/crd2pulumi-darwin-amd64/crd2pulumi ./crd2pulumi
+```
+
+## Running the App (TypeScript)
+
+Follow the steps in [Pulumi Installation and Setup](https://www.pulumi.com/docs/get-started/install/) and [Configuring Pulumi
+Kubernetes](https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/setup/) to get setup with Pulumi and Kubernetes.
+
+Install dependencies:
+
+```bash
+$ cd kubernetes-ts-crontabs
+$ npm install
+```
+
+Create a new stack:
+
+```bash
+$ pulumi stack init dev
+```
+
+At first, the provided `index.ts` program shouldn't run, since we haven't actually generated the `./crontabs` code yet.
+
+> `resourcedefinition.yaml` is a k8s CRD that specifies a CronTab CustomResource. It's also used as an example in the [Kubernetes Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
+
+Generate the strongly-typed CronTab resource in the current directory:
+```bash
+$ ../crd2pulumi nodejs ../resourcedefinition.yaml .
+```
+
+This should generate a `./crontabs` folder, where we can import the useful classes `v1.CronTab` and `NewCronTabDefinition`.
+> This saves us a lot of time, since we can create the CRD in a single line and get strongly-typed arguments in the CustomResource. If you're curious, the comments contain the code that we would've written without `crd2pulumi`.
```

Comment:
Blank line

```suggestion

> This saves us a lot of time, since we can create the CRD in a single line and get strongly-typed arguments in the CustomResource. If you're curious, the comments contain the code that we would've written without `crd2pulumi`.
```